packages/server/routes/admin.js

```diff
@@ -886,9 +875,25 @@ router.get(
                   ),
                   tr(th(req.__("Node.js version")), td(process.version)),
                   tr(
-                    th(req.__("Database")),
+                    th(req.__("Database type")),
                     td(db.isSQLite ? "SQLite " : "PostgreSQL ", dbversion)
                   ),
+                  tr(
```

Comment:
I don't think it is a good idea to expose host, port and user (schema is fine). On some platforms (e.g. Heroku) these are secret and would expose this information to sub-tenant owners, allowing them to further probe the DB host.

Reply:
Yes, better to show this only for main site for admin users only. I'll update this now

Reply:
Updated.